lib/omnibus/packagers/xz.rb

```diff
@@ -27,12 +27,15 @@ class Packager::XZ < Packager::Base
 
     build do
       out_file = windows_safe_path(Config.package_dir, archive_name)
-      out_source_path = "#{windows_safe_path(project.install_dir)}/*"
+      input_paths = "#{windows_safe_path(project.install_dir)}/*"
+      project.extra_package_files.each do |file|
+        input_paths = "#{input_paths} #{file}"
+      end
```

Comment:
Splitting the suggestion in two parts.  I'd say this would look a bit cleaner using `.join(" ")` rather than an ad-hoc loop.

```suggestion
      input_paths = ["#{windows_safe_path(project.install_dir)}/*"] + project.extra_package_files
```